components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/DeviceDAO.java

```diff
@@ -62,8 +62,18 @@
 
     /**
      * Get the count of devices
+     *
      * @return device count
      * @throws DeviceManagementDAOException
      */
     int getDeviceCount() throws DeviceManagementDAOException;
+
+    /**
+     * Get the list of devices that matches with the given device name.
+     *
+     * @param deviceName Name of the device
```

Comment:
These comments have been removed in commit [1].
[1] https://github.com/wso2/carbon-device-mgt/commit/8d348a358c8af9d8b9a5f0cb1cc1dd9aca13fbdb